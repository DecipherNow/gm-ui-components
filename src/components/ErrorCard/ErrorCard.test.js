import React from "react";
import { shallow } from "enzyme";

import { ErrorCard } from "../";

describe("ErrorCard", () => {
  it("matches snapshot", () => {
    const anErrorCard = shallow(<ErrorCard />);
    expect(anErrorCard).toMatchSnapshot();
  });
  it("renders an error message", () => {
    const aErrorCard = shallow(<ErrorCard errorMsg="Woopsie" />);
    expect(aErrorCard).toMatchSnapshot();
  });
});
