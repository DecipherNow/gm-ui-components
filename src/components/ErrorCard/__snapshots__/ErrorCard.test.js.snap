// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorCard matches snapshot 1`] = `
<ErrorBox>
  <ErrorGraphic />
  <ErrorMessage>
    Error
  </ErrorMessage>
  <ErrorDetail />
</ErrorBox>
`;

exports[`ErrorCard renders an error message 1`] = `
<ErrorBox>
  <ErrorGraphic />
  <ErrorMessage>
    Woopsie
  </ErrorMessage>
  <ErrorDetail />
</ErrorBox>
`;
